iFixit/react-commerce
# Align favicon with ifixit.com

closes #454 

This PR substitutes the project specific favicon with the set specified in #454

## QA
1. Visit the Vercel PR preview
2. Verify that favicon is now consistent with the one on ifixit.com
